README.md

```diff
@@ -1 +1,103 @@
-# building2parcel-pix2pix
\ No newline at end of file
+# building2parcel-pix2pix
+
+This repository contains the implementation of CycleGAN and Pix2Pix models for image-to-image translation tasks. It provides comprehensive setup guides for both Windows and Linux environments.
+
+## Project Overview
+
+This project focuses on implementing and training CycleGAN and Pix2Pix models for various image-to-image translation tasks. These models can be used for a wide range of applications, including style transfer, object transfiguration, season transfer, and photo enhancement.
+
+## Environment Setup
+
+Choose the appropriate guide based on your operating system:
+
+- [Windows Setup Guide](./docs/Windows_README.md)
+- [Linux Setup Guide](./docs/Linux_README.md)
+
+## Quick Start
+
+1. Clone this repository:
+
+   ```
+   git clone https://github.com/scalable-design-participation-lab/re-blocking.git
+   cd re-blocking
+   ```
+
+2. Add and initialize the pix2pix submodule:
+
+   ```
+   git submodule add https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix.git
+   git submodule update --init --recursive
+   ```
+
+   This project uses the original [pix2pix repository](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix) as a submodule. Adding it as a submodule allows us to:
+
+   - Keep the original implementation separate from our project-specific code.
+   - Easily update to new versions of pix2pix when needed.
+   - Clearly distinguish between the original code and our modifications.
+
+3. Follow the setup instructions for your operating system (see links above).
+
+4. Prepare your dataset as described in the setup guides.
+
+5. Train the model:
+
+   - For Windows, use the `[train_pix2pix].bat` script
```

Comment:
We might also have to add documentation of how to run it without the `.bat` and `bash` files